dropshot/src/server.rs

```diff
@@ -806,6 +807,23 @@ async fn http_request_handle_wrap<C: ServerContext>(
     #[cfg(feature = "usdt-probes")]
     let local_addr = server.local_addr;
 
+    // In the case the client disconnects early, the scopeguard allows us
+    // to perform extra housekeeping before this task is dropped.
+    let on_disconnect = guard((), |_| {
+        warn!(request_log, "client disconnected before response returned");
```

Comment:
Being really nitty, but that's because these log messages are most often used when debugging really tricky issues so I want to make sure they're precise about what they say:

```suggestion
        warn!(request_log, "request handling cancelled (client disconnected)");
```

Could you add `latency_us` to this log message as well?  Similar to L840:

```rust
let latency_us = start_time.elapsed().as_micros();
```

and then include that in the log message.

Reply:
Not nitty at all, makes perfect sense. Done 👍